src/one_dragon/base/controller/pc_controller_base.py

```
@@ -566,15 +576,35 @@ def get_mouse_sensitivity():
 
 
 def drag_mouse(start: Point, end: Point, duration: float = 0.5):
-    """按住鼠标左键进行画面拖动。
+    """按住鼠标左键进行画面拖动 (消除拖动后的惯性)。
 
     Args:
         start: 原位置
         end: 拖动位置
         duration: 拖动鼠标到目标位置，持续秒数
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)